### ISSUE TITLE
downsampling error Windows-specific (similar to issue 64)

### ISSUE THREAD

**Issue by beefpepsi** [nolanlab/spade#87]
Hello, 

I work for FlowJo and recently started to learn Spade after receiving some customer questions. While the issues reported by our customers typically have to do with R not being in path, :) , I did find something today worth a comment. 

Whenever I try to invoke the command 

SPADE.driver(data_file_path, out_dir=output_dir, transforms=transforms, cluster_cols=c("PE-A", "PE-Cy55-A", "PE-Cy7-A", "APC-Ax700-A"))

where the variables have been set identically, and pointing to the same data file, I get a different output depending if I'm on  OSX vs Windows 7. Same version of R, same version of Spade. I'm not doing this from FlowJo but the R interface.  

On OSX this command correctly produces a bunch of outputs in a subfolder. 

On windows, I get console errors similar to issue#64 (null values in keywords), but I think the issue is actually in the process of creating the downsampled FCS file.

In short, given the same input + commands, I get a different "downsampled" result, and execution halts at that point on windows, but finishes OK on OSX. I've used cytof data, diva data, and some old FCS2 cellquest data, all producing same difference. 

Have you seen anything like this, and if so, how do you get around it? I admit it could be user error 99% likely, 

thank you

maciej simm
flowjo
